content/news/2019-02-15-scalatra-2-6-5-released.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Scalatra 2.6.5 is out"
+layout: news
+author: Naoki Takezoe
+twitter: takezoen
```

Comment:
Why aren't `author` and `twitter` yours? :-)

Reply:
oh! I'll fix it